src/main/php/CommandLine/StaticCodeAnalysis/PHPCodeSniffer/TerminalCommand.php

```diff
@@ -16,9 +16,10 @@
 use Zooroyal\CodingStandard\CommandLine\StaticCodeAnalysis\Generic\TerminalCommand\Fix\FixTrait;
 use Zooroyal\CodingStandard\CommandLine\StaticCodeAnalysis\Generic\TerminalCommand\Multiprocess\MultiprocessTerminalCommand;
 use Zooroyal\CodingStandard\CommandLine\StaticCodeAnalysis\Generic\TerminalCommand\Multiprocess\MultiprocessTrait;
+// phpcs:ignore -- I did not find a way to either break this line or to make it shorter.
+use Zooroyal\CodingStandard\CommandLine\StaticCodeAnalysis\Generic\TerminalCommand\PhpVersion\MinimalVersionDependantTerminalCommand;
```

Comment:
this would be possible, but does not quite make it more readable I think :sweat_smile: 
```php
use Zooroyal\CodingStandard\CommandLine\StaticCodeAnalysis\Generic\TerminalCommand\PhpVersion\{
    MinimalVersionDependantTerminalCommand,
    MinimalVersionDependentTrait,
    PhpVersionConverter
};
```

Reply:
yeah. thought about that but came to the same conclusion as you did. :/